content/en/docs/how-tos/interact-with-running-jobs.md

```diff
@@ -0,0 +1,146 @@
+
+**Note:** Authentication to the OpenShift cluster is delegated to GitHub and requires that you are a member of the
+`OpenShift` organization.
+
+From GitHub, you can access the build logs from the `Details` link next to each job listed in the checkbox at the bottom of the PR description
+page. This gives an overall picture of the test output, but you might want to follow each job and test more closely while it runs. In that case,
+you can access the CI cluster console. From the job `Details`, grep for this line near the top of the `Build Logs` to locate the cluster/project
+where your test is running:
+
+```bash
+2021/02/04 04:37:08 Using namespace https://console.build02.ci.openshift.org/k8s/cluster/projects/ci-op-mtn6xs34
+```
+As the PR author, you are the administrator of the project (`ci-op-mtn6xs34` in example above). You will therefor have access to
+the link above. From the console, to login, choose `github`. Once in the console, you can follow the logs from the running pods _or_
+you can grab the login command from the upper right `?` -> `Command Line Tools` -> `Copy Login Command`. Again, choose the `github` 
+authentication, then `Display Token`. Copy in your terminal to access this single project of the CI cluster. Before running the login
+command, you might run `unset KUBECONFIG` if you currently have an active cluster & local kubeconfig, since this login command will either
+update the currently set KUBECONFIG or write/update to ~/.kube/config.
+
+As the project administrator, you may give access to your project to other members of the GitHub OpenShift organization. Afer logging into the
+project with `oc`, use this command to give other members access to the project running your PR:
+
+```bash
+$ oc adm policy add-role-to-user admin <github_user>
+```
+
+**Note:** Shortly after your job completes whether due to success or failure, the project as well as the external cluster launched from
+this project will be torn down and terminated.
+
+## Where are the Logs?
+
+From GitHub, you can access the build logs from the `Details` link next to each job listed in the checkbox at the bottom of the PR description
+page. This gives an overall picture of the test output, but you might want to follow each job and test more closely while it runs. In that case,
+from the CI cluster console accessed above, you have a choice between `Administrator` and `Developer` menu. Choose `Administrator` if it is not
+already chosen. You can access the `pods` running the tests from `Workloads` -> `Pods` and then the `Logs` tab across the top. Pods
+usually have multiple containers, and each container can be accessed from the dropdown menu above the logs terminal.
+This is equivalent to running the following from your local terminal, if you are currently logged into the CI cluster:
+
+```bash
+$ oc project ci-op-xxxxxxx
+$ oc get pods
+$ oc logs -f <pod-name> -c <container-name>
+```
+
+## Access the terminal of the pod Running the tests
```

Comment:
```suggestion
## Access the terminal of the pod running the tests
```

Reply:
: ) I've updated and made these all TitleCase but yea, that was a mistake (from going back and forth to 'should I or should I not make them TitleCase) lol